selftests/functional/test_output.py

```diff
@@ -391,11 +379,11 @@ def test_redirect_output(self):
     @unittest.skipIf(perl_tap_parser_uncapable(),
                      "Uncapable of using Perl TAP::Parser library")
     def test_tap_parser(self):
-        perl_script = script.TemporaryScript("tap_parser.pl",
```

Comment:
Yep, hang on, I'm trying to convince travis... I kept both there to give you the credit you deserve (I'd not be able to write the perl script).